inc/lift/Request.h

```diff
@@ -228,38 +252,52 @@ class Request {
     /**
      * @return  the number of connections made to make this request
      */
+     [[nodiscard]]
     auto GetNumConnects() const -> uint64_t;
     
     /**
      * Set the verify behavior of the CURLOPT_SSL_VERIFYPEER on the curl_handle
-     * @param verify the verify value to set the CURLOPT_SSL_VERIFYPEER option to
+     * @param verify Bool indicating whether we should require cURL to verify the SSL peer (true) or not (false)
      */
-    auto SetVerifySSLPeer(int64_t verify) -> void;
+    auto SetVerifySSLPeer(bool verify) -> void;
 
     /**
      * Set the verify behavior of the CURLOPT_SSL_VERIFYHOST on the curl_handle
-     * @param verify the verify value to set the CURLOPT_SSL_VERIFYHOST option to
+     * @param verify Bool indicating whether we should require cURL to verify the SSL host (true) or not (false)
+     */
+    auto SetVerifySSLHost(bool verify) -> void;
+    
+    /**
+     * Tells cURL to using an Accept-Encoding header that includes all built-in
+     * supported encodings in a comma-separated list.
+     * IMPORTANT: Using this is mutually exclusive with adding your own Accept-Encoding header.
      */
-    auto SetVerifySSLHost(int64_t verify) -> void;
+    auto SetAcceptAllEncoding() -> void;
 
     /**
      * Resets the request to be re-used.  This will clear everything on the request.
      */
     auto Reset() -> void;
-
 private:
     /**
      * Private constructor -- only the RequestPool can create new Requests.
      * @param request_pool The request pool that generated this handle.
      * @param url          The url for the request.
-     * @param timeout      The timeout for the request in milliseconds.
+     * @param connection_timeout The maximum time to wait before quitting, calling the
+     *          on_complete_handler and closing the connection.
```

Comment:
Indent looks odd. Align with description start on previous line?
Or is this standard practice?